examples/shouldi/tests/test_dependency_check.py

```diff
@@ -0,0 +1,35 @@
+import pathlib
+
+from dffml.util.os import prepend_to_path
+from dffml.util.net import cached_download_unpack_archive
+from dffml.util.asynctestcase import AsyncTestCase
+
+from shouldi.dependency_check import run_dependency_check
+
+
+class TestRunDependencyCheckOp(AsyncTestCase):
+    @cached_download_unpack_archive(
+        "https://download.java.net/openjdk/jdk14/ri/openjdk-14+36_linux-x64_bin.tar.gz",
+        pathlib.Path(__file__).parent / "downloads/java.tar.gz",
```

Comment:
Please make sure all of these use the `pathlib` way of joining paths

```suggestion
        pathlib.Path(__file__).parent / "downloads" / "java.tar.gz",
```